sound/soc/bcm/hifiberry_digi.c

```diff
@@ -202,7 +214,9 @@ static int snd_rpi_hifiberry_digi_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id snd_rpi_hifiberry_digi_of_match[] = {
+	{ .compatible = "geekaroo,wm8804-digi", },
 	{ .compatible = "hifiberry,hifiberry-digi", },
+	{ .compatible = "iqaudio,wm8804-digi", },
```

Comment:
Adding compatible strings for known users seems wrong to me - this is now meant to be generic driver. Either add a second vendor-neutral compatible string or make all overlays use "hifiberry,hifiberry-digi", which seems like a reasonable tariff for using the Hifiberry driver.